trin-validation/src/accumulator.rs

```diff
@@ -37,11 +43,8 @@ pub struct MasterAccumulator {
 impl MasterAccumulator {
     /// Load default trusted master acc
     pub fn try_from_file(master_acc_path: PathBuf) -> anyhow::Result<MasterAccumulator> {
-        let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
-        path.push(master_acc_path);
-        let raw = std::fs::read(&path)
-            .map_err(|_| anyhow!("Unable to find master accumulator at path: {:?}", path))?;
-        MasterAccumulator::from_ssz_bytes(&raw)
+        let raw = TrinValidationAssets::get(&(*master_acc_path).display().to_string()[..]).unwrap();
```

Comment:
Let's handle this unwrap too, and map the error to anyhow (example in the line below). We don't want naked unwraps in our code, except for tests.

Reply:
resolved